app-modules/prospect/src/Filament/Resources/ProspectResource/Pages/ManageProspectAlerts.php

```diff
@@ -157,6 +157,7 @@ public function table(Table $table): Table
             ])
             ->headerActions([
                 CreateAction::make()
+                    ->authorize('create', $this->getOwnerRecord())
```

Comment:
Does `getOwnerRecord` reference the correct Model here?

Reply:
Yep ! It's referring correct model here.